deployments/operator/device/dsa/dsa.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: inteldeviceplugins-controller-manager
+  namespace: inteldeviceplugins-system
```

Comment:
was the namespace required for this to work?

Reply:
I have tried to omit it, however kustomize errors without namespace:
```
Error: no matches for IdId apps_v1_Deployment|~X|inteldeviceplugins-controller-manager; failed to find unique target for patch apps_v1_Deployment|inteldeviceplugins-controller-manager
```